tests/test_pipeline_big.py

```diff
@@ -0,0 +1,487 @@
+            h5f.close()
+        else:
+            message_str = f"File name with {h5_file_name} string cannot be found."
+            raise FileNotFoundError(message_str)
+
+    residual_im = data_gt - data_result
+    res_norm = np.linalg.norm(residual_im.flatten()).astype("float32")
+    assert res_norm < 1e-6
+
+
+# This is data from Christina Reinhard. Stitching is problematic with it
+# @pytest.mark.full_data
+# def test_pipeline_gpu_360_paganin_FBP_i13_180622_preview(
+#     get_files: Callable,
+#     cmd,
+#     i13_180622,
+#     gpu_pipeline_360_paganin_FBP,
+#     output_folder,
+# ):
+#     # NOTE that the intermediate file with file-based processing will be saved to /tmp
+
+#     _change_value_parameters_method_pipeline(
+#         gpu_pipeline_360_paganin_FBP,
+#         method=[
+#             "standard_tomo",
+#             "normalize",
+#             "find_center_360",
+#             "find_center_360",
+#             "find_center_360",
+#             "find_center_360",
+#             "find_center_360",
+#             "sino_360_to_180",
+#             "FBP",
+#         ],
+#         key=[
+#             "preview",
+#             "minus_log",
+#             "ind",
+#             "win_width",
+#             "side",
+#             "norm",
+#             "use_overlap",
+#             "rotation",
+#             "center",
+#         ],
+#         value=[
+#             {"detector_y": {"start": 300, "stop": 600}},
+#             False,
+#             "mid",
+#             50,
+#             0,
+#             True,
+#             True,
+#             "left",
+#             None,
+#         ],
+#     )
+
+#     cmd.pop(4)  #: don't save all
+#     cmd.insert(5, i13_180622)
+#     cmd.insert(7, gpu_pipeline_360_paganin_FBP)
+#     cmd.insert(8, output_folder)
+
+#     subprocess.check_output(cmd)
+
+#     files = get_files("output_dir/")
+
+#     #: check the generated reconstruction (hdf5 file)
+#     h5_files = list(filter(lambda x: ".h5" in x, files))
+#     assert len(h5_files) == 1
+
+# TODO re-enable this
+# # load the pre-saved numpy array for comparison bellow
+# data_gt = gpu_diad_FBP_k11_38731_npz["data"]
+# axis_slice = gpu_diad_FBP_k11_38731_npz["axis_slice"]
+# (slices, sizeX, sizeY) = np.shape(data_gt)
+
+# step = axis_slice // (slices + 2)
+# # store for the result
+# data_result = np.zeros((slices, sizeX, sizeY), dtype=np.float32)
+
+# path_to_data = "data/"
+# for file_to_open in h5_files:
+#     if "httomolibgpu-FBP" in file_to_open:
+#         h5f = h5py.File(file_to_open, "r")
+#         index_prog = step
+#         for i in range(slices):
+#             data_result[i, :, :] = h5f[path_to_data][:, index_prog, :]
+#             index_prog += step
+#         h5f.close()
+#     else:
+#         raise FileNotFoundError("File with httomolibgpu-FBP string cannot be found")
+
+# residual_im = data_gt - data_result
+# res_norm = np.linalg.norm(residual_im.flatten()).astype("float32")
+# assert res_norm < 1e-6
```

Comment:
While I think TODOs only existing in code comments isn't ideal, if that's the preferred way of keeping track of tasks then fair enough.

However, I think this one is more than just a bit questionable: it's not in a test function, it's at the module-level. If uncommented I don't know if this would get picked up correctly by pytest if it's being executed at the module-level, and is easily mistakable as scratch work. If this TODO needs to be kept in, could it be put inside a test function please?

Reply:
sorry it got confusing due to indentation. It is a part of the test above, which I decided to comment due to data being a bit weird one and not a representative case. I'll probably delete this test and references to this data altogether as I added more sensible data sample from the same beamline.